### PR TITLE
[Cleanup] Fix Android Studio java warnings in anki/servicelayer #13282

### PR DESCRIPTION
**Pull request by henriqueccmac**
## Purpose / Description
Get to know the project. Fixed some android studio warnings and weak warnings as per #13282.

## Fixes
AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/scopedstorage/MigrateEssentialFiles.kt: fixed typos in references to functions and constants.
AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/scopedstorage/MoveDirectory.kt: fixed import paths and made a function private.
AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/scopedstorage/MoveFileOrDirectory.kt: removed unused import.
AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/scopedstorage/migrateuserdata/MigrateUserData.kt: made relevant functions private.

## Checklist
_Please, go through these checks before submitting the PR._

- [X ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] You have commented your code, particularly in hard-to-understand areas
- [X] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by david-allison — APPROVED** [review]
Cheers!

**Review by david-allison — CHANGES_REQUESTED** [review]
Tests are failing

```
> Task :AnkiDroid:compilePlayDebugUnitTestKotlin FAILED
e: file:///home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/test/java/com/ichi2/anki/servicelayer/scopedstorage/MoveDirectoryTest.kt:22:66 Unresolved reference: MoveDirectory
e: file:///home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/test/java/com/ichi2/anki/servicelayer/scopedstorage/MoveFileOrDirectoryTest.kt:19:66 Unresolved reference: MoveDirectory
```

File: AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/scopedstorage/MoveDirectory.kt [hunk 1]
```diff
@@ -14,14 +14,14 @@
  *  this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-package com.ichi2.anki.servicelayer.scopedstorage.migrateuserdata
+package com.ichi2.anki.servicelayer.scopedstorage
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/scopedstorage/MoveDirectory.kt:17]: issue is likely here - I suspect we want to keep the package and move the file

Comment (henriqueccmac) [reply AnkiDroid/src/main/java/com/ichi2/anki/servicelayer/scopedstorage/MoveDirectory.kt]: I agree. I moved the file and changed the imports accordingly.

**Review by david-allison — APPROVED** [review]
Squash merge and force push on your first change. Impressive!

Thanks so much, welcome to AnkiDroid :)

**Pull request merged**